demes/man/load_deme.Rd

```diff
@@ -0,0 +1,17 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
This file looks autogenerated.

Reply:
The .Rd files are documentation source files for R packages. They autogenerated because this package is using roxygen2 to manage documentation, as is the standard today. The .Rd files should be included in the package source as users don't run roxygen2. Neither does CRAN.

Reply:
Ah, so these files don't get rebuilt when the package is installed? I guess that makes sense, because it avoids a dependency on roxygen2.